common/lib/xmodule/xmodule/templates/problem/customgrader.yaml

```diff
@@ -4,11 +4,31 @@ metadata:
     markdown: !!null
 data: |
       <problem>
+      <p>    
+      In custom Python-evaluated input (also called “write-your-own-grader”
```

Comment:
is this the advanced "CUSTOM PYTHON-EVALUATED INPUT" problem type? When I add it to the snadbox, I get this error:  
Problem i4x://ManTestX/ManTest1/problem/e2fdb8fb2db2497db327c1471999edd3 has an error:
cannot create LoncapaProblem i4x://ManTestX/ManTest1/problem/e2fdb8fb2db2497db327c1471999edd3: attributes construct error, line 60, column 21